languages/swift/concepts/generic-functions/about.md

```diff
@@ -0,0 +1,96 @@
+One of the most important concepts in programming is code reuse. One should be able to limit the number of times they write a piece of code in order to make the code smaller and easier to understand, while at the same time lowering the amount of time and effort needed to write a program.
+
+We've seen one of the common ways this is accomplished in programming, functions. While functions are occasionally used just to repeat the same code over and over, more often than not, they are executed with different input values. Structs, classes, and enums also allow us to reuse code involving both methods (functions) and properties (data), reusing the code in those structs, classes, and enums with many different values.
```

Comment:
The documents should be considered as stand-alone documents, so I would suggest to remove the "we've" seen bit.